libcrux-ml-kem/Cargo.toml

```diff
@@ -31,6 +31,7 @@ libcrux-platform = { version = "0.0.2", path = "../sys/platform" }
 libcrux-sha3 = { version = "0.0.2", path = "../libcrux-sha3" }
 libcrux-intrinsics = { version = "0.0.2", path = "../libcrux-intrinsics" }
 hax-lib.workspace = true
+libcrux-secrets = { version = "0.0.2-beta.3", path = "../secrets" }
```

Comment:
This version doesn't seem right? The workspace currently has `0.0.2`. But you may want to start the secrets crate off with `0.0.1-beta.1` or something like that.

Reply:
If we are merging this to libcrux-main, do we need a non-beta version?

Reply:
I switched to 0.0.2, but we can change it if needed. I don't think there is anything very controversial in this version of secrets.